section3/exercises/arrays.js

```diff
@@ -96,21 +104,27 @@ YOU DO:
 Using the variables defined below, write a program that will tell a user if they
 will be able to call an Uber.
 
-The user can call an uber if they have 15% battery remaining, or more. In this case, it doesn't matter 
+The user can call an uber if they have 15% battery remaining, or more. In this case, it doesn't matter
 if the user has a charger at all, or what type.
 The can call an uber if they have a charger and it is a car charger.
 
 */
-var percentBatteryLeft = 12;
+var percentBatteryLeft = 15;
 var hasCharger = true;
 var chargerType = "car";
 
 // Write your conditional here
-
+if (percentBatteryLeft >= 15){ // Can call an Uber if battery life is greater or equal to 15
+  console.log("You are able to call an Uber, hooray!");
+} else if (hasCharger === true && chargerType === "car") { // Can call Uber if percentBatteryLeft is less than 15, but has a car charger
+    console.log("You are able to call an Uber, hooray!");
```

Comment:
Be sure to keep your indentation consistent, this line looks further indented line 118.